src/common/perf_counters.h

```diff
@@ -233,6 +233,9 @@ class PerfCounters
 
   ~PerfCounters();
 
+  bool time_filtered = false;
+  ceph::coarse_real_clock::duration time_alive;
```

Comment:
@cbodley I gave this a try and it didn't work out, but trying lead me to thinking about the interface we want to provide and what the config variables that might apply to these values mean. 

Under what you propose if time_alive is set to 0 than labeled perf counters would always be dumped if the conditional in perf_counters.cc, line 143

```
// top commented out line is what it currently is
//if ((*l)->time_filtered && ((*l)->last_updated + (*l)->time_alive < ceph::coarse_real_clock::now())) {
if ((*l)->time_alive.count() && ((*l)->last_updated + (*l)->time_alive < ceph::coarse_real_clock::now())) {
    // don't dump this counter
    continue;
}
```

However if I think about it, `time_alive` is supposed to be the number of seconds the counters can be dumped for, if the value is 0, that means they should never be dumped. If you only left the comparison of time alive + last_update and the current time, then by default no labeled counters would ever get dumped because the comparison in that conditional would always be true since time_alive would default to 0.

So what should a `time_alive` of 0 and any correspond config variable like `rgw op counters dump expiration` mean? That counters never get dumped when that is the case because they don't have a time window to be dumped? Maybe we should keep the `bool` and have that correspond to a config variable itself just so that things would be clear to the user